simplified-books-api/src/main/java/org/nypl/simplified/books/api/BookmarkKind.kt

```diff
@@ -31,7 +31,7 @@ sealed class BookmarkKind(
    */
 
   object ReaderBookmarkExplicit :
-    BookmarkKind("https://www.w3.org/ns/oa#bookmarking") {
+    BookmarkKind("http://www.w3.org/ns/oa#bookmarking") {
```

Comment:
wondering if it might be better to have a constant for this fixed value somewhere, unless.... this is actually it (did I mention I don't know kotlin?)

Reply:
That's actually the constant there.